package.json

```diff
@@ -23,55 +23,55 @@
     "url": "https://github.com/ooyala/html5-skin.git"
   },
   "dependencies": {
-    "classnames": "*",
+    "classnames": "^2.2.6",
     "create-react-class": "^15.6.3",
     "deepmerge": "^1.4.0",
     "lodash.debounce": "4.0.3",
     "lodash.values": "4.1.0",
-    "prop-types": "^15.6.1",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1",
+    "prop-types": "^15.6.2",
+    "react": "^16.5.2",
+    "react-dom": "^16.5.2",
     "react-scrollbar": "^0.5.4",
     "screenfull": "3.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0-beta.51",
-    "@babel/preset-env": "^7.0.0-beta.51",
-    "@babel/preset-react": "^7.0.0-beta.51",
+    "@babel/core": "^7.1.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
```

Comment:
Should we update this to 7.1.0 as well? To keep in sync with the other babel presets.

Reply:
done